src/caustics/lenses/base.py

```diff
@@ -129,13 +129,18 @@ def forward_raytrace(
             Tensor of source redshifts.
         params: (Packed, optional)
             Dynamic parameter container for the lens model. Defaults to None.
+
         epsilon: Tensor
             maximum distance between two images (arcsec) before they are considered the same image.
+        *Unit: arcsec*
+
         n_init: int
             number of random initialization points used to try and find image plane points.
         fov: float
             the field of view in which the initial random samples are taken.
 
+        *Unit: radians*
```

Comment:
The spacing doesn't match the rest of the usages.

Reply:
Updated